docs/pl/grm.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
this file should be replaced by docs/grm.html?lang=pl

Reply:
This file will replased this one.
https://crowdin.com/project/moneymanagerex/pl#
Now is very important to move English files into 'en' folder, then upload them into Crowbin.

Reply:
It is easier to replace the file than to alter the script. Fake file is workaround (temporary solution).